e2e/cli-e2e/mocks/fs.mock.ts

```diff
@@ -27,3 +28,15 @@ export function cleanFolderPutGitKeep<T extends object>(
     }
   }
 }
+
+export function setupFolder<T extends object>(
+  dirName = 'tmp',
+  content?: { [key in keyof T]: string },
+) {
```

Comment:
There's no advantage to using generics here. It has no effect on any other parameter or on what is returned.

```suggestion
export function setupFolder(
  dirName = 'tmp',
  content?: Record<string, string>,
) {
```